docs/cfg/buildprofiles.xml

```diff
@@ -10,7 +10,9 @@
         </browser-edits-url-override>
         <kotlin-latest-url>%kotlinLatestUrl%</kotlin-latest-url>
         <generate-only-default-anchors>true</generate-only-default-anchors>
-        <web-root>https://www.jetbrains.com/idea/</web-root>
+        <web-root>https://kotlinlang.org/</web-root>
```

Comment:
I guess there should be `https://kotlinlang.org/docs/` ?

Reply:
According to the WH team announcement:

> web-root— path to help instance, everything that precedes product web-path. For example, all instances hosted on jetbrains.com must specify web-root as https://www.jetbrains.com/help

Reply:
resolved